client.go

```diff
@@ -140,6 +144,10 @@ func (client *Client) SetImageFromBytes(data []byte) error {
 	}
 
 	img := C.CreatePixImageFromBytes((*C.uchar)(unsafe.Pointer(&data[0])), C.int(len(data)))
+	if img == nil {
+		return fmt.Errorf("failed to create PixImage from bytes")
```

Comment:
Same as above, it would be more friendly if we can add some information to the error message.
For example, how about `len(data)`?

Reply:
I've updated this error message now to include the number of bytes as suggested

Reply:
Let me check if you pushed your commit to include the update?

Reply:
Not sure why that didn't get pushed, have pushed it now regardless